src/main/java/net/dv8tion/jda/core/handle/PresenceUpdateHandler.java

```diff
@@ -158,7 +168,37 @@ protected Long handleInternally(JSONObject content)
             else
             {
                 //In this case, this PRESENCE_UPDATE is for a Relation.
+                if (api.getAccountType() != AccountType.CLIENT)
+                    return null;
+                JDAClient client = api.asClient();
+                FriendImpl friend = (FriendImpl) client.getFriendById(userId);
 
+                if (friend == null)
+                {
+                    if (status != OnlineStatus.OFFLINE)
+                        return null;
```

Comment:
What is the point of this?

Reply:
ugh, looks like I forgot to remove this after I decided not to cache these presence updates